src/components/modal/ModalAddPatient.tsx

```diff
@@ -56,46 +66,82 @@ const ModalAddPatient: React.FC<ModalAddPatientProps> = ({
               onChange={handleChangeNewPatient}
             />
           </Form.Group>
+          {/* Accordion for additional details */}
+          <Accordion defaultActiveKey=''>
+            <Accordion.Item eventKey='0'>
+              <Accordion.Header>{t('form.additionalDetails')}</Accordion.Header>
+              <Accordion.Body>
+                <Form.Group as={Col} className='mt-2'>
+                  <Form.Label><FontAwesomeIcon icon={['fas', 'address-card']} /> {t('form.address')}</Form.Label>
+                  <Form.Control
+                    name='street'
+                    type='text'
+                    value={patientState.street}
+                    onChange={handleChangeNewPatient}
+                  />
+                </Form.Group>
+                <Form.Group as={Col} className='mt-2'>
+                  <Form.Label className='me-2'><FontAwesomeIcon icon={['fas', 'calendar-days']} /> {t('form.birthday')}</Form.Label>
+                  <ReactDatePicker
+                    selected={birthdayDate}
+                    onChange={onBirthdayChange}
+                    dateFormat='dd/MM/yyyy'
```

Comment:
reuse from a const to be defined somewhere

Reply:
done